frontend/src/main.ts

```diff
@@ -26,7 +26,8 @@ Vue.use(Vuetify, {
   icons: {
     listening: "fas fa-headphones",
     track: "fas fa-cut",
-    quality: "fas fa-check"
+    quality: "fas fa-check",
+    musicNotes: "fas fa-music"
```

Comment:
The name of the custom icon should be semantic. See the previous ones. I propose renaming this to `sound`.